package.json

```diff
@@ -28,11 +28,22 @@
     ],
     "all": true,
     "cache": true,
-    "check-coverage": true
+    "check-coverage": true,
+    "exclude": [
+      "coverage/**",
+      "test{,s}/**",
+      "**/.{prettier,eslint,mocha}rc.{js,cjs}"
+    ]
   },
   "scripts": {
+    "format": "prettier . --write",
+    "lint:formatting": "prettier . --check",
+    "lint:js": "eslint .",
+    "lint": "npm-run-all --parallel lint:*",
+    "pretest": "npm run lint",
```

Comment:
Let's remove the `pretest` lifecycle command and replace with a "lint" task, [like so](https://github.com/stylelint/stylelint-config-standard/blob/e1c9d96c554da7b5f887036c6cf43fa5639de3b1/.github/workflows/nodejs.yml#L15-L42) as we only want lint once on the CI. This will be consistent with other repositories.

Reply:
I removed the 'pretest' script and added lint to CI.